### PR TITLE
Check the origin cluster status

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@GowthamShanmugam , @SanjalKatiyar Please review

File: packages/mco/components/modals/app-failover-relocate/target-cluster-selector.tsx [hunk 1]
```diff
@@ -221,18 +221,21 @@ export const TargetClusterSelector: React.FC<TargetClusterSelectorProps> = ({
...
     if (isClusterAvailable) {
       // Check DR fencing status for origin cluster
       const drClusters = drClusterList.map((drCluster) => drCluster);
+      // Get the origin/primary cluster from the fence list
+      const originCluster = drClusters.find(
+        (drCluster) => getName(drCluster) !== getName(managedCluster)
+      );
       const replicationType = getRelicationTypeUsingDRClusters(drClusters);
       if (replicationType === REPLICATION_TYPE.SYNC) {
+        // Validate origin cluster and its availablity
```
Comment (GowthamShanmugam) [inline packages/mco/components/modals/app-failover-relocate/target-cluster-selector.tsx:234]: After discussing with the DR team. Here expected change is:

- Failover: Ensure From cluster is fenced, and To cluster is NOT fenced
- Relocate: Ensure From and To clusters are NOT fenced

Also, modify the error message accordingly.

Comment (SanjalKatiyar) [reply packages/mco/components/modals/app-failover-relocate/target-cluster-selector.tsx:234]: @TimothyAsirJeyasing update the PR as per this comment and add BZ to the title as well...

Comment (SanjalKatiyar) [reply packages/mco/components/modals/app-failover-relocate/target-cluster-selector.tsx:234]: run i18n command

**Pull request closed without merging**